src/yelp-application.c

```diff
@@ -117,6 +117,49 @@ struct _YelpApplicationPrivate {
 };
 
 static void
+open_uri (YelpApplication *app,
+          YelpUri         *uri,
+          gboolean         new_window,
+          gboolean         fallback_help_list);
+
+static void 
+activate_show_page (GSimpleAction *action,
+                    GVariant      *parameter,
+                    gpointer       user_data)
+{
+    YelpApplication *app = YELP_APPLICATION (user_data);
+    YelpUri *uri = yelp_uri_new (DEFAULT_URI);
+    gchar *xref = g_strconcat ("xref:", g_variant_get_string (parameter, NULL), NULL);
+    YelpUri *page_uri = yelp_uri_new_relative (uri, xref);
+
+    open_uri (app, page_uri, TRUE, FALSE);
+
+    g_object_unref(uri);
+    g_object_unref(page_uri);
+    g_free(xref);
+}
+
+static void
+activate_show_search (GSimpleAction *action,
+                      GVariant      *parameter,
+                      gpointer       user_data)
+{
+    YelpApplication *app = YELP_APPLICATION (user_data);
+    YelpUri *uri = yelp_uri_new (DEFAULT_URI);
+    YelpUri *page_uri = yelp_uri_new_search (uri, g_variant_get_string (parameter, NULL));
+
+    open_uri (app, page_uri, TRUE, FALSE);
+
+    g_object_unref(uri);
```

Comment:
You just wait and see - you'll start adding those spaces in another project where they don't want those spaces. ;)